labelbox/data/annotation_types/metrics/scalar.py

```diff
@@ -28,6 +40,14 @@ class ScalarMetric(BaseMetric):
     value: Union[ScalarMetricValue, ScalarMetricConfidenceValue]
     aggregation: ScalarMetricAggregation = ScalarMetricAggregation.ARITHMETIC_MEAN
 
+    @validator('metric_name')
+    def validate_metric_name(cls, name: str):
+        clean_name = name.lower().strip()
+        if name.lower().strip() in RESERVED_METRIC_NAMES:
+            raise ValueError(f"`{clean_name}` is a reserved metric name. "
+                             "Please provide another value for `metric_name`.")
+        return clean_name
```

Comment:
I think we shouldn't force all lowercase, it's just for validation purpose

Reply:
I think we should be consistent, if we're validating against lower-cased value then we should store it that way as well

Reply:
but on the other hand it will break existing metrics, yeah 🤔

Reply:
I'd say validating against lower-cased metric name feels weird, we should just trim it

Reply:
@msokoloff1 @kozikkamil wdyt?

Reply:
alright, I think that's not so important

now we return the source name as it's currently in prod version 👌